app/client/sagas/authSagas.ts

```diff
@@ -251,6 +259,14 @@ function* refreshToken() {
     if (token_request.auth_token) {
       storeSessionToken(token_request.auth_token);
 
+      let orgId = getOrgIds();
+      let orgIds = orgId && orgId.split(",");
+      if (orgIds && orgIds.length > 1) {
+        token_request["organisation_ids"] = orgIds;
+      } else {
+        token_request["organisation_ids"] = null;
+      }
```

Comment:
This made the most sense to me. As `saveOrgId` stores org ids in the web browser local storage, it made sense on reauth to check if multi org is active when reauth occurs